macros/base/start_print.cfg

```diff
@@ -403,14 +403,39 @@ gcode:
     {% endif %}
 
 
-[gcode_macro _MODULE_CUSTOM1]
+[gcode_macro _MODULE_EXTRUDER_HEATING_TAP]
 gcode:
-    # ---- CUSTOM Macro section
-    # this section is reserved for personal customized start actions, which can be combined with all other start actions
-    # in order to use this, create a new macro in overrides.cfg
-    # [gcode_macro _MODULE_CUSTOM1]
-    # gcode:
-    #   ## Your custom code here
+    # Heat the nozzle to tap probing temperature ontop of the purge bucket (if available)
+    # or in the middle of the bed and wait 30s to let the nozzle dilate and stabilize
+    {% set EXTRUDER_TEMP = printer["gcode_macro _USER_VARIABLES"].tap_max_probing_temp|float %}
+    {% set INITIAL_TOOL = printer["gcode_macro START_PRINT"].initial_tool %}
+
+    {% set verbose = printer["gcode_macro _USER_VARIABLES"].verbose %}
+    {% set status_leds_enabled = printer["gcode_macro _USER_VARIABLES"].status_leds_enabled %}
+    {% set ercf_enabled = printer["gcode_macro _USER_VARIABLES"].ercf_enabled %}
```

Comment:
for what do you need ercf here?

Reply:
I added a proper and full review, So indeed this line is not needed, but there is more to remove. Please see below (I'll solve this conversation to keep thing in the full review and avoid dispersion :) )